src/Core/Grid/Query/AttachmentQueryBuilder.php

```diff
@@ -43,24 +43,24 @@ final class AttachmentQueryBuilder extends AbstractDoctrineQueryBuilder
     /**
      * @var string
      */
-    private $employeeIdLang;
+    private $contextLanguageId;
 
     /**
      * @param Connection $connection
      * @param string $dbPrefix
      * @param DoctrineSearchCriteriaApplicatorInterface $searchCriteriaApplicator
-     * @param string $employeeIdLang
+     * @param string $contextLanguageId
      */
     public function __construct(
         Connection $connection,
         string $dbPrefix,
         DoctrineSearchCriteriaApplicatorInterface $searchCriteriaApplicator,
-        string $employeeIdLang
+        string $contextLanguageId
```

Comment:
Actually, while we're fixing this you should inject the `LanguageContext` instead of the ID

Reply:
@jolelievre done